WebExample/tests/styles.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {test, expect} from '@playwright/test';
+import type {Page} from '@playwright/test';
+import * as TEST_CONST from '../../testConstants';
+import {setupInput} from './utils';
+
+const testMarkdownContentStyle = async ({styleName, style, page}: {styleName: string; style: string; page: Page}) => {
+  const inputLocator = await setupInput(page);
+
+  const elementHandle = inputLocator.locator('span', {hasText: styleName}).last();
+
+  let elementStyle;
+
+  if (elementHandle) {
+    await elementHandle.waitFor({state: 'attached'});
+
+    elementStyle = await elementHandle.getAttribute('style');
+  }
+  expect(elementStyle).toEqual(style);
+};
+
+test.beforeEach(async ({page}) => {
+  await page.goto(TEST_CONST.LOCAL_URL, {waitUntil: 'load'});
+  await page.click('[data-testid="reset"]');
+});
+
+test.describe('markdown content styling', () => {
+  test('bold', async ({page}) => {
+    await testMarkdownContentStyle({styleName: 'bold', style: TEST_CONST.MARKDOWN_STYLE_DEFINITIONS.bold.style, page});
+  });
+
+  test('link', async ({page}) => {
+    await testMarkdownContentStyle({styleName: 'link', style: TEST_CONST.MARKDOWN_STYLE_DEFINITIONS.link.style, page});
+  });
+
+  test('title', async ({page}) => {
```

Comment:
Please make sure that the names of the tests are aligned with the rest of the codebase
```suggestion
  test('h1', async ({page}) => {
```